src/twisted/mail/test/test_pop3.py

```diff
@@ -1,3 +1,4 @@
+# -*- test-case-name: twisted.mail.pop3 -*-
```

Comment:
:-D I wasn't sure at all with this one, this needs to be better in the Wiki! What does it do when properly set up anyway?